lib/srv/reexec.go

```diff
@@ -93,6 +94,18 @@ type ExecCommand struct {
 
 	// IsTestStub is used by tests to mock the shell.
 	IsTestStub bool `json:"is_test_stub"`
+
+	// RemoteAddr is the address of the remote host.
+	RemoteAddr [4]int32 `json:"remote_addr"`
+
+	// The hostname of the node.
+	Hostname string `json:"hostname"`
+
+	// The path of the system utmp database.
+	UtmpPath *string `json:"utmp_path"`
+
+	// The path of the system wtmp log.
+	WtmpPath *string `json:"wtmp_path"`
```

Comment:
Any reason these are pointers?

Also Go comments start with the name of the variable, so for this (and elsewhere):

```
// UtmpPath is the path to the system utmp database.
```

Reply:
They're pointers because they're optional, if not provided, uacc will default to system defaults.

Reply:
Adjusted comments